libs/entry-components/styles/modules/components/dialogs/_generator.scss

```diff
@@ -2,12 +2,12 @@
 
 @mixin generate-from($theme) {
 	.entry-dialog {
-		.title {
+		.dialog-header .title {
```

Comment:
is it possible to put the title inside dialog-header, with that will be 3 nesting levels, or I'm wrong?

Reply:
It's 3 levels one way or another :)